packages/edit-site/src/components/page-pages/index.js

```diff
@@ -157,6 +158,20 @@ export default function PagePages() {
 		totalPages,
 	} = useEntityRecords( 'postType', postType, queryArgs );
 
+	useEffect( () => {
```

Comment:
What is this code doing?

Reply:
I added a comment.

Reply:
Yes if we follow the current direction of being the consumer managing the selection state, the consumer needs this logic.